scripts/vlsi_rom_gen

```diff
@@ -23,10 +23,10 @@ module {name}(
   reg [{output_width_minus_1}:0] rom [0:{depth_minus_1}];
 
 
-  initial begin
+  initial begin: init_and_load
     integer i;
     // 256 is the maximum length of $readmemh filename supported by Verilator
-    reg [255:0] [7:0] path;
+    reg [255*8:0] path;
```

Comment:
Wait, actually, I'm going to revoke my approval. This should be 256*8 - 1, no?

Reply:
yes you're right ... I thought I had rationalized this in my head, but incorrectly